File: Realtimemonitoring/Monitoring.Web.Contract/Models/Outcoming/Response/ResponseTemplate.cs
using System.Runtime.Serialization;

namespace Sychev.Monitoring.Web.Contract.Models.Outcoming.Response
{
    /// <summary>
    /// 
    /// </summary> 
    [DataContract]
    public class ResponseTemplate
    {
        /// <summary>
        /// Success
        /// </summary>
        [DataMember]
        public bool Success { get; set; }

        /// <summary>
        /// Message
        /// </summary>
        [DataMember]
        public string Message { get; set; }
    }
}

File: Realtimemonitoring/Src/Monitoring.Web/Scripts/TS/Incoming/Responce/ResponseTemplate.ts
module Main {
    export class ResponceTemplate {
        constructor() {

        }

        private _success: boolean;

        get Success(): boolean {
            return this._success;
        }
        set Success(success: boolean) {
            this._success = success;
        }


        private _message: string;

        get Message(): string {
            return this._message;
        }
        set Message(message: string) {
            this._message = message;
        }
    }
} 